main.py

```diff
@@ -0,0 +1,423 @@
+from fastapi import FastAPI, HTTPException, Path, Body
+from fastapi.encoders import jsonable_encoder
+from typing import Annotated
+from models.Dispatcher import Dispatcher, UpdateDispatcherModel
+from models.Repair import RepairModel
+from models.Trailer import Trailer, UpdateTrailerModel
+from models.Truck import Truck, UpdateTruckModel
+from models.Manager import Manager, UpdateManagerModel
+from get_db_pymongo import get_database
+from bson import ObjectId
+from models.TruckDriver import TruckDriver, UpdateTruckDriverModel
+from utils import serialize_collection
+
+app = FastAPI()
+
+db = get_database()
+
+@app.get("/")
+async def root():
+    return {"message": "Hello World"}
+
+@app.get("/trucks/")
+async def get_trucks():
+    try:
+        collection = db.get_collection("trucks")
+        trucks = list(collection.find())
+        
+        return {"serialized_trucks": serialize_collection(trucks)}
```

Comment:
We never return an object, that has a key: "serialized_trucks"

We do: 

```suggestion
        return serialize_collection(trucks)
```